internal/cmd/testdata/scripts/templatefuncs.txt

```diff
@@ -27,10 +27,14 @@ cmp stdout golden/include-relpath
 chezmoi execute-template '{{ joinPath "a" "b" }}'
 stdout a${/}b
 
-# test lookPath template function
+# test lookPath template function to find in PATH
 chezmoi execute-template '{{ lookPath "go" }}'
 stdout go$exe
 
+# test lookPath template function to check if file exists
+chezmoi execute-template '{{ lookPath "/non-existing-file" }}'
+stdout ''
```

Comment:
This check isn't quite correct: `stdout` matches the output with a regular expression, and the empty regular expression matches everything. Instead this line should be:
```
! stdout .
```
to check that the output is empty.

Reply:
Thank you very much for this. Indeed I didn't search, I just putted whatever came to my mind first :)

Reply:
Well, it does not seem to work. :(

Reply:
`stdout '^$'` did the trick.